packages/components/package.json

```diff
@@ -42,8 +42,14 @@
 		"@emotion/styled": "^11.11.0",
 		"@wordpress/base-styles": "^5.8.0",
 		"@wordpress/components": "^28.8.0",
+		"@wordpress/compose": "7.8.0",
+		"@wordpress/date": "5.8.0",
+		"@wordpress/html-entities": "4.8.0",
+		"@wordpress/i18n": "^5.8.0",
 		"@wordpress/icons": "^10.8.0",
+		"@wordpress/primitives": "^4.8.0",
 		"@wordpress/react-i18n": "^4.8.0",
+		"@wordpress/url": "^4.8.0",
```

Comment:
I know that those are the same versions used across the repo, but is it expected that `@wordpress/*` packages are not aligned on the same version?

Reply:
It's possible to happen, and sometimes intentionally. Ideally, they should all be up to date, and using the same version, though. And the work to update to the latest version is almost done, see https://github.com/Automattic/wp-calypso/pull/96470